Cargo.toml

```
@@ -9,13 +9,14 @@ license = "MIT"
 
 [features]
 default = ["embed-dll"]
-embed-dll = []
+embed-dll = ["memory-module-sys"]
 
 [dependencies]
 winapi = { version = "0.3.8", features = ["combaseapi", "libloaderapi"] }
 widestring = "0.4.0"
 # com = "0.1"
 com = { git = "https://github.com/microsoft/com-rs.git" }
+memory-module-sys = { git="https://github.com/indygreg/rs-memory-module-sys", branch = "main", optional = true }
```

Use the crates.io version?

```suggestion
memory-module-sys = { version = "0.3.0", optional = true }
```